scripts/build-images

```diff
@@ -16,9 +16,9 @@ xargs -n1 -t docker image pull --quiet << EOF > build/images.txt
     docker.io/rancher/k8s-metrics-server:v0.3.6
     docker.io/rancher/klipper-helm:v0.3.0
     docker.io/rancher/kube-proxy:${KUBERNETES_VERSION}
-    k8s.gcr.io/defaultbackend-amd64:1.5
-    k8s.gcr.io/pause:3.2
-    quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.30.0
+    docker.io/rancher/pause:3.2
+    docker.io/rancher/nginx-ingress-controller-defaultbackend:1.5-rancher1
+    docker.io/rancher/nginx-ingress-controller:nginx-0.30.0-rancher1
```

Comment:
It seems like we've tried to setup a pattern for the image name + tag but applied it inconsistently.

Reply:
Yeah, this is an existing image that's used by RKE1. See the rant here: https://github.com/rancher/image-mirror/pull/26#discussion_r479576501